huaweicloud/services/dms/data_source_huaweicloud_dms_rocketmq_instances.go

```diff
@@ -229,14 +230,14 @@ func DmsRocketMQInstancesInstanceRefSchema() *schema.Resource {
 	return &sc
 }
 
-func DmsRocketMQInstancesInstanceRefCrossVpcInfoRefSchema() *schema.Resource {
+func RocketMQInstancesInstanceRefCrossVpcInfoRefSchema() *schema.Resource {
```

Comment:
```suggestion
func rocketMQInstancesInstanceRefCrossVpcInfoRefSchema() *schema.Resource {
```
it seems it should be a private function

Reply:
done